File: csharp/gabaritos/gabarito17.cs
using System;

class Retangulo
{
    public double Largura { get; set; }
    public double Altura { get; set; }

    public Retangulo(double largura, double altura)
    {
        Largura = largura;
        Altura = altura;
    }

    public double Area()
    {
        return Largura * Altura;
    }

    public double Perimetro()
    {
        return 2 * (Largura + Altura);
    }
}

class Program
{
    static void Main()
    {
        Console.Write("Digite a largura: ");
        double largura = double.Parse(Console.ReadLine());
        Console.Write("Digite a altura: ");
        double altura = double.Parse(Console.ReadLine());
        Retangulo ret = new Retangulo(largura, altura);
        Console.WriteLine($"Área: {ret.Area()}");
        Console.WriteLine($"Perímetro: {ret.Perimetro()}");
    }
}

File: cpp/gabaritos/gabarito17.cpp
#include <iostream>

class Retangulo {
private:
    double largura, altura;
public:
    Retangulo(double l, double a) : largura(l), altura(a) {}
    double area() {
        return largura * altura;
    }
    double perimetro() {
        return 2 * (largura + altura);
    }
};

int main() {
    double largura, altura;
    std::cout << "Digite a largura: ";
    std::cin >> largura;
    std::cout << "Digite a altura: ";
    std::cin >> altura;
    Retangulo ret(largura, altura);
    std::cout << "Área: " << ret.area() << std::endl;
    std::cout << "Perímetro: " << ret.perimetro() << std::endl;
    return 0;
}